packages/account/src/Components/file-uploader-container/file-uploader-utils.js

```diff
@@ -1,13 +1,13 @@
 import { localize } from '@deriv/translations';
-import { compressImg, convertToBase64, isImageType, getFormatFromMIME } from '_common/image_utility';
+import { compressImg, convertToBase64, isImageType, getFormatFromMIME } from 'Duplicated/_common/image_utility';
 
-export const compressImageFiles = files => {
+export const compressImageFiles = (files) => {
```

Comment:
Wouldn't `prettier` remove the parenthesis, why re-add the parenthesis? Parenthesis is only necessary if there is more than one parameter as I recall we're using something like `as-needed` for the `arrow-parens` rule in our `prettier` config.

Reply:
@zaki-binary @reza-fs prettier 2.0 changes this behavior (and accounts is using prettier > 2.0 whereas our other packages are using 1.x)

so either update the .prettierrc or downgrade the version to be consistent with other packages

Reply:
@reza-fs @oskar-binary @zaki-binary I think we should update our other prettiers then, because this is better behaviour.

Reply:
@reza-fs but not in this PR, let's do it in another PR.